pkg/xds/server/dataplane_status_tracker_test.go

```diff
@@ -183,7 +206,8 @@ var _ = Describe("DataplaneStatusTracker", func() {
 			// when
 			discoveryRequest := &envoy.DiscoveryRequest{
 				Node: &envoy_core.Node{
-					Id: "default.example-001",
+					Id:       "default.example-001",
+					Metadata: version,
```

Comment:
Could you set this a little bit more manually?
```
			Metadata: &pstruct.Struct{
				Fields: map[string]*pstruct.Value{
					"dataplaneTokenPath": &pstruct.Value{
						Kind: &pstruct.Value_StringValue{
							StringValue: "/tmp/token",
						},
					},
				},
			},
```
reference pkg/xds/server/dataplane_metadata_tracker_test.go:19

To be honest I do not fully trust that we have a "version" branch with such serializing

Reply:
You're right. It didn't work. Thank you. I fixed it and added more test cases.